internal/services/account_service.go

```diff
@@ -8,6 +8,30 @@ import (
 	"github.com/go-chi/render"
 )
 
+func CreateAccount(w http.ResponseWriter, r *http.Request) {
```

Comment:
Exposing account creation via API is not needed, however, we can implement it just for the sake of better administration options in case we need to add one explicitly.

However, what is very important is that if we start adding actions other than get/list then we must implement admin permission. I created a ticket for that: https://issues.redhat.com/browse/HMSCONTENT-123